docs/DeveloperGuide.md

```diff
@@ -242,6 +243,49 @@ Step 3. The parser will construct the `DeleteTagCommand` differently based on th
 
 Step 4. During `DeleteTagCommand#execute()`, the force flag is checked. If present, it unassigns all `Person` with the `Tag` tag1 before deleting tag1.
 
+### Tag Feature
+
+The Tag Feature allows users to store additional details of a Person in WedLinker.
+Tags would support the following functions:
+
+* `Create Tag` — Creates a `Tag` in WedLinker to be used as additional information of any `Person`.
+* `Tag` — Assigns `Tag` to a `Person.
+* `Untag` — Unassigns a `Tag` from a `Person`.
+* `Delete Tag` — Deletes the `Tag` from WedLinker.
+
+#### Implementation
+
+Given below is an example usage scenario and how Tags are used in WedLinker.
+
+Step 1. The user launches the application, `Tags` are loaded into the `Model`.
+
+Step 2. The user executes `create-tag t/Photographer`. WedLinker will create a Tag based on the name provided. In this case: `Photographer`.
```

Comment:
I think this should include a step where `create-tag` checks for the existence of the tag in the system already